### ISSUE TITLE
[BUG] Error: invalid parameters: parameter aad_tenant_id not defined in bundle

### ISSUE THREAD

**Issue by guybartal** [microsoft/AzureTRE#993]
**Describe the bug**
running command ```make porter-install DIR=./templates/workspace_services/guacamole``` while following [Guacamole service bundle docs](https://github.com/microsoft/AzureTRE/blob/develop/docs/tre-templates/workspace-services/guacamole.md) results with an error:
```
Resolving parameter sources...
No parameter sources defined!
Error: invalid parameters: parameter aad_tenant_id not defined in bundle
make: *** [Makefile:169: porter-install] Error 1
```

**Steps to reproduce**

1. follow all steps in  [Guacamole service bundle docs](https://github.com/microsoft/AzureTRE/blob/develop/docs/tre-templates/workspace-services/guacamole.md) 
1. See error

**Comment by marrobi** [comment]
@anatbal for Guacamole the terraform auth AAD tenant might be different from the ARM/Azure subscription AAD tenant. So need to be careful.

**Comment by marrobi** [comment]
@anatbal lets discuss tomorrow - I might be getting confused, or might be fixed by @eladiw when he looks at the tokens being used by Guacamole.

**Comment by marrobi** [comment]
This will be resolved by #983 

https://github.com/microsoft/AzureTRE/blob/b544826e53a44409a458b3956d59ab78e2c15502/templates/workspace_services/guacamole/terraform/locals.tf#L9
